test/data/extensions/theme/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
copyright?

Reply:
Hmm, not sure we can add comments to json file type.